rpm/util-linux.spec

```diff
@@ -348,6 +354,11 @@ exit 0
 # FIXME: Remove after UsrMove
 %attr(4755,root,root)	%{_bindir}/umount
 %attr(4755,root,root)	%{_bindir}/su
+# Symlinks for systemd
+/sbin/agetty
```

Comment:
Please put them into the usrmerged section.
They are redundant after the usrmerge.